### PR TITLE
Support for Swift 3

### PR DESCRIPTION
**Pull request by hm-91**
Code updated to fix Swift 3 issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This basically is a Swift version of the iOS client found in this tutorial: ["Ho
 
 ### How to Try Out
 
- 1. Clone the repo: e.g. `git clone https://github.com/hideya/loopback-swift-simple-example`
+ 1. Clone the repo: e.g. `https://github.com/hm-91/loopback-swift-simple-example.git`
```
Comment (hideya) [inline README.md]: Ah, putting the URL there was not good an idea as it has the dependency to the project owner's id.
How about just saying"1. Close the repo" ?

**Review by hideya — COMMENTED** [review]
Thanks for doing this!  I have only one comment.

**Comment by hideya** [comment]
@Haari one more actually -- would you mind to update the following code fragment in README.md too?

``` objc
let adapter = LBRESTAdapter(URL: NSURL(string: "http://localhost:3000/api"))
```

**Comment by hm-91** [comment]
@hideya Changes has been done. Thank you for pointing me out.

**Comment by hideya** [comment]
@hm-91 PR landed.  Thank you for your prompt action!

**Comment by hideya** [comment]
@hm-91 Ahhh, one more I missed...  Could you please take care of updating the following line in README.md and make another PR?

``` swift
let coffeeShopRepo = adapter.repositoryWithClass(CoffeeShopRepository) as! CoffeeShopRepository
```

**Pull request merged**